frontend/make-env.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
```

Comment:
Was thinking... is this script needed for development? Or only on CI?

If only on CI, it could exit early and not write the .env file... by checking if the "`CI`" env var is set.

Reply:
It's needed in both.